x-pack/plugins/enterprise_search/server/plugin.ts

```diff
@@ -41,6 +41,7 @@ import { appSearchTelemetryType } from './saved_objects/app_search/telemetry';
 import { registerTelemetryUsageCollector as registerASTelemetryUsageCollector } from './collectors/app_search/telemetry';
 import { registerEnginesRoute } from './routes/app_search/engines';
 import { registerCredentialsRoutes } from './routes/app_search/credentials';
+import { registerLogSettingsRoutes } from './routes/app_search/log_settings';
```

Comment:
Just curious - do we need to specifically call this out as `log_settings` vs. just `settings`? I think I have a slight preference to simplifying this to just 

```suggestion
import { registerSettingsRoutes } from './routes/app_search/settings';
```

I think that's fine because on the off-chance we add more non-ILM/log settings in the future that call different API endpoints, we can just continue to add more `router` APIs to the same file, and it's not a _huge_ deal because they conceptually fall under the same "Settings" bucket.

Reply:
@constancecchen Yeah I think that's a good suggestion, I'll change this file to be "settings"

Reply:
361cfea075a725dbcc18babb2ef7737e934380cc